CaloCluster/fcl/prolog.fcl

```diff
@@ -25,11 +28,12 @@ CaloTrigger : {
     minSeedAmplitude      : 433
     minEnergy             : 50
     timeCorrection        : 20.0
-    adcToEnergy           : 0.0461333
+    adcToEnergy           : 0.0459634#0.0461333
```

Comment:
Hi Sophie - I just noticed this now, after everything is merged in.  I have a strong preference against recording old values in comments - that's what we have a code management system for.  Please fix this sort of thing as targets of opportunity when you touch the code in the future.

Reply:
ah, sorry, must have missed that, I thought I'd removed it. Will be sure to correct in future.